external/CMakeLists.txt

```diff
@@ -39,23 +39,21 @@ endif()
 # --------------------------------------------------------------------------------------------------
 # zenoh
 
-set(ZENOHC_CMAKE_ARGS -DZENOHC_CARGO_CHANNEL=+stable -DZENOHC_BUILD_WITH_SHARED_MEMORY=TRUE)
```

Comment:
do we want the unstable api? what does it enable?

Reply:
cache publisher and being able to get the list of topic and